python/tank/platform/qt/aspect_preserving_label.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2013 Shotgun Software Inc.
```

Comment:
We’ll need to keep it in because I expect different apps and engines will want to customize the dialog with a different bitmap (Desktop for example would want a different bitmap). Unless the resolution of the image is also part of the contract with core, we’ll have to provide the aspect preserving bit. Which to me sounds reusable, because this means we don’t have to resize all our assets.